src/main/java/usecase/StartingPointUseCase.java

```diff
@@ -8,8 +8,10 @@
  * Represent the use case when the player passing through the starting point
  */
 public class StartingPointUseCase {
-    public static void giveBonus(Player player, StartingPoint startingPoint){
-        int bonus = startingPoint.getBonus();
+    public static void giveBonus(Player player){
+        int bonus = StartingPoint.getBonus();
```

Comment:
the `bonus` should be initialized by the constructor. What will happen when calling `StartingPoint.getBonus()`? It seems that it would return 0?